app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss

```diff
@@ -1,25 +1,231 @@
 @import "govuk/components/accordion/accordion";
```

Comment:
Looks like we've got no instances of `govuk-frontend` accordion styles in the markup anymore. Can this import be removed?